pkg/kind/cluster.go

```diff
@@ -108,11 +110,25 @@ func (c *Cluster) getConfig() ([]byte, error) {
 		}
 	}
 
+	registryConfig := ""
+	for _, s := range c.registryConfig {
+		path := os.ExpandEnv(s)
+		if _, err := os.Stat(path); err == nil {
+			registryConfig = path
+			break
+		}
+	}
+
+	if len(c.registryConfig) > 0 && registryConfig == "" {
+		setupLog.Info("--registry-config flag used but no registry config was found")
```

Comment:
Do you think it's a good idea to fail here? We think the user's intention was to mount registry configuration, but it does not exist. If we do not fail, the cluster is created with configuration that does not match what the user wanted it to be. So it might be a better idea to fail early and ask the user for a valid configuration.

Reply:
Yeah I think that's a good idea, i threw a `os.Exit(1)` in that block

Reply:
I think it's better to return an error and let the error cascade out to cobra to have an unified experience for users. Right now all fatal errors go out like that.